internal/lsp/server.go

```diff
@@ -837,14 +858,6 @@ func (l *LanguageServer) StartCommandWorker(ctx context.Context) { //nolint:main
 				if err != nil {
 					fmt.Fprintf(os.Stderr, "failed to evaluate workspace path: %v\n", err)
 
-					cleanedMessage := strings.Replace(err.Error(), l.workspaceRootURI+"/", "", 1)
```

Comment:
This is also removed since ANY error is reported below and this just makes it a duplicated message.